templates/legal/ubuntu-advantage-service-description/ja/index.md

```diff
@@ -12,6 +12,8 @@ context:
 
 {% include "shared/pricing/_ua-for-infrastructure-ja.html" %}
 
+<!-- TODO is the new table ok for the legal section? + Needs Japanese translation -->
```

Comment:
I don't think questions should be in the code base.

I see 4 TODOs in the code. Why is it? Are you still waiting for answers? Or are we planning to address them later? If the plan is to deal with them later then we should instead have an issue explaining what is needed instead of comments.

Reply:
This PR going on a feature branch it was more to keep track of what was missing. I planned to make sure all the todos were gone before merging to master. You're right I should create issues!